src/index.html

```diff
@@ -12,7 +12,55 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="base__body">
+    <div>
```

Comment:
The `div` tag on this line is a container for all your stars elements. It would be more semantic to use a `section` tag instead of a `div` tag here. `section` is a more descriptive tag and helps to organize the document into discrete sections, which can be useful for accessibility tools and SEO.